AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_EntityReferenceAdded.py

```diff
@@ -119,33 +148,109 @@ def AtomEditorComponents_EntityReference_AddedToEntity():
         general.idle_wait_frames(1)
         Report.result(Tests.creation_redo, entity_reference_entity.exists())
 
-        # 5. Enter/Exit game mode.
+        # Entities for EntityIdReferences tests
+        test_1 = EditorEntity.create_editor_entity('test_1')
+        test_2 = EditorEntity.create_editor_entity('test_2')
+        test_3 = EditorEntity.create_editor_entity('test_3')
+
+        # 5. 'EntityIdReferences' is a container property
+        Report.result(
+            Tests.entity_id_references_is_container,
+            entity_reference_component.is_property_container(
+                AtomComponentProperties.entity_reference('EntityIdReferences')))
+
+        # 6. Append item to 'EntityIdReferences'
+        entity_reference_component.append_container_item(AtomComponentProperties.entity_reference('EntityIdReferences'),
+                                                         test_1.id)
+        Report.result(
+            Tests.container_append,
+            entity_reference_component.get_container_item(
+                AtomComponentProperties.entity_reference('EntityIdReferences'), 0) == test_1.id)
+
+        # 7. Add item to 'EntityIdReferences'
+        entity_reference_component.add_container_item(AtomComponentProperties.entity_reference('EntityIdReferences'),
+                                                      1, test_1.id)
+        Report.result(
+            Tests.container_add,
+            entity_reference_component.get_container_count(
+                AtomComponentProperties.entity_reference('EntityIdReferences')) == 2
+        )
+
+        # 8. Update item in 'EntityIdReferences'
+        entity_reference_component.update_container_item(AtomComponentProperties.entity_reference('EntityIdReferences'),
```

Comment:
here and throughout same styling nitpick: i would prefer 2 lines where the second line contains all of the parameters:
```
entity_reference_component.update_container_item(
    AtomComponentProperties.entity_reference('EntityIdReferences'), 1, test_2.id)
```
But I will mention (since its semi-related) that for the `Report.result()` and `Report.critical_result()` calls the extra lines are good for the `Tests` value and the value it compares against for the test. The reason I like the different formatting for these calls is its not dealing with the `Report` or `Tests` objects. Just wanted to mention that while adding this feedback.

Reply:
fixing style